flaml/autogen/agent/human_agent.py

```diff
@@ -0,0 +1,53 @@
+from .agent import Agent
+from flaml.autogen.code_utils import extract_code, execute_code
+
+
+class HumanProxyAgent(Agent):
+    """(Experimental) A proxy agent for human, that can execute code and provide feedback to the other agents."""
+
+    DEFAULT_SYSTEM_MESSAGE = """You are human agent. You can execute_code or give feedback to the sender.
+    """
+
+    def __init__(
+        self, name, system_message="", work_dir=None, interactive_mode=True, is_termination_msg=None, **config
```

Comment:
add docstr. Revise the choices of interactive model if the current choices are confusing.